app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def av_actor_age
```

Comment:
Super nitpicky, but I'd name this `average_actor_age` so there no guessing about what "av" means. In larger and more complex projects, you aren't going to want to spend the mental energy to figure out other developers' abbreviations.